src/cls/cephfs/cls_cephfs.cc

```diff
@@ -123,6 +123,70 @@ static int accumulate_inode_metadata(cls_method_context_t hctx,
   return 0;
 }
 
+// I want to select objects that have a name ending 00000000
+// and an xattr (scrub_tag) not equal to a specific value.
+// This is so special case that we can't really pretend it's
+// generic, so just fess up and call this the cephfs filter.
```

Comment:
I think this is fine, but we could also have a generic "name and xattr" filter. Require it to match the name in a (definable, or just search-based) pattern, and to either match or not match an xattr name and value.